arch/xtensa/src/esp32s3/Kconfig

```diff
@@ -366,26 +366,26 @@ config ESP32S3_I2S0_SAMPLE_RATE
 config ESP32S3_I2S0_BCLKPIN
 	int "I2S0 BCLK pin"
 	default 4
-	range 0 33 if ESP32S3_I2S0_ROLE_MASTER
-	range 0 39 if ESP32S3_I2S0_ROLE_SLAVE
+	range 0 48 if ESP32S3_I2S0_ROLE_MASTER
+	range 0 48 if ESP32S3_I2S0_ROLE_SLAVE
```

Comment:
merge to `range 0 48`

Reply:
Sorry, I could not understand. Shall I create a different pull request for the range 0-48?

Reply:
both ESP32S3_I2S0_ROLE_MASTER and ESP32S3_I2S0_ROLE_SLAVE have the same range in your patch, why not just change to `range 0 48` without if condition?